server/src/main/java/com/sauron/account/BankAccount.java

```diff
@@ -1,38 +1,53 @@
-package com.sauron.user;
+package com.sauron.account;
 
+import javax.persistence.CollectionTable;
 import javax.persistence.Column;
+import javax.persistence.ElementCollection;
 import javax.persistence.Entity;
 import javax.persistence.EnumType;
 import javax.persistence.Enumerated;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.Table;
+import java.util.Collection;
 import java.util.Objects;
 
 @Entity
-@Table(name = "BANK_ACCOUNT_MAPPING")
-public class BankAccountMapping {
+@Table(name = "BANK_ACCOUNT")
+public class BankAccount {
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @Column(unique = true, updatable = false, nullable = false)
     private Long id;
 
+    @Column(nullable = false, name = "USER_ID")
+    private Long userId;
+
     @Column(nullable = false)
     private String name;
 
     @Column(name = "MARKER_COLOR", nullable = false)
     @Enumerated(EnumType.STRING)
     private Color markerColor;
 
-    public BankAccountMapping() {
+    @ElementCollection
+    @CollectionTable(
+            name = "BANK_CONNECTION_DATA",
+            joinColumns = @JoinColumn(name = "BANK_ACCOUNT_ID")
+    )
+    private Collection<BankConnectionData> bankConnectionData;
+
+    public BankAccount() {
     }
 
-    public BankAccountMapping(Long id, String name, Color markerColor) {
+    public BankAccount(Long id, String name, Color markerColor, Collection<BankConnectionData> bankConnectionData) {
```

Comment:
In this constructor userId is missing. Does it make sense to create BankAccount without any user assigned to it?